Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
The source archive tarball is only 22M. And it should let us update the formula with `brew bump-formula-pr` since the URL has the version number in it. Also, it has the license file, changelog, etc. which get installed automatically brew in the prefix, so it's not a total waste as long as you leave it unpacked in the buildpath, and do the other operations in a subdirectory and/or in the cache.